doc/releases/migration-guide-3.6.rst

```diff
@@ -212,6 +212,10 @@ Bluetooth
   Any pointer to a UUID must be prefixed with `const`, otherwise there will be a compilation warning.
   For example change ``struct bt_uuid *uuid = BT_UUID_DECLARE_16(xx)`` to
   ``const struct bt_uuid *uuid = BT_UUID_DECLARE_16(xx)``. (:github:`66136`)
+* The :c:func:`bt_l2cap_chan_send` API now doesn't return the number of bytes sent.
```

Comment:
This should go in the change log. Also add that it never returned anything useful.